### PR TITLE
Fix DeprecationWarning on autocompletion with jedi 0.17.0

### PR DESCRIPTION
**Pull request by gorogoroumaru**
with jedi 0.17.0

```
In [1]: import warnings
In [2]: warnings.filters.remove(('ignore', None, DeprecationWarning, None, 0))
In [3]: a=0
In [4]: a.<TAB>
```

throws a deprecation warning below

DeprecationWarning: Providing the column is now done in the functions themselves like `Script(...).complete(line, column)`

So, I fixed it.

### CODE REVIEW

File: IPython/core/completer.py [hunk 1]
```diff
@@ -1371,15 +1371,14 @@ def _jedi_matches(self, cursor_column:int, cursor_line:int, text:str):
...
-            text[:offset], namespaces, column=cursor_column, line=cursor_line + 1)
+        script = jedi.Interpreter(text[:offset],namespaces)
         try_jedi = True
 
         try:
             # find the first token in the current tree -- if it is a ' or " then we are in a string
             completing_string = False
             try:
-                first_child = next(c for c in interpreter._get_module().tree_node.children if hasattr(c, 'value'))
+                first_child = next(c for c in script._get_module().tree_node.children if hasattr(c, 'value'))
```
Comment (LucianaMarques) [inline IPython/core/completer.py:1381]: If you keep `interpreter` you don't have to change the following lines.

**Review by LucianaMarques — COMMENTED** [review]
Thanks for the PR! It's just now clear to me why your changes fixed the problem. Could you please give us some running examples? 

(I'm unable at this moment to run the project, so that would help me)

File: IPython/core/completer.py [hunk 2]
```diff
@@ -1371,15 +1371,14 @@ def _jedi_matches(self, cursor_column:int, cursor_line:int, text:str):
                 else:
                     raise ValueError("Don't understand self.omit__names == {}".format(self.omit__names))
 
-        interpreter = jedi.Interpreter(
-            text[:offset], namespaces, column=cursor_column, line=cursor_line + 1)
+        script = jedi.Interpreter(text[:offset],namespaces)
```
Comment (LucianaMarques) [inline IPython/core/completer.py:1374]: Why did you change the name here from `interpreter` to `script`?

Comment (gorogoroumaru) [reply IPython/core/completer.py:1374]: Sorry, it’s my mistake.
I will fix it.

**Comment by gorogoroumaru** [comment]
Running Examples

IPython 8.0.0.dev

[before]
In [1]: import jedi

In [2]: jedi.__version__
Out[2]: '0.17.0'

In [3]: import warnings

In [4]: warnings.filters.remove(('ignore', None, DeprecationWarning, None, 0))

In [5]: a=0

In [6]: a.<tab>
\Anaconda3\envs\jedi\lib\site-packages\jedi-0.17.0-py3.8.egg\jedi\api\__init__.py:841: DeprecationWarning: Providing the line is now done in the functions themselves like `Script(...).complete(line, column)`
  super(Interpreter, self).__init__(code, environment=environment,
\Anaconda3\envs\jedi\lib\site-packages\jedi-0.17.0-py3.8.egg\jedi\api\__init__.py:841: DeprecationWarning: Providing the column is now done in the functions themselves like `Script(...).complete(line, column)`
  super(Interpreter, self).__init__(code, environment=environment,

[after]
In [1]: import jedi

In [2]: jedi.__version__
Out[2]: '0.17.0'

In [3]: import warnings

In [4]: warnings.filters.remove(('ignore', None, DeprecationWarning, None, 0))

In [5]: a=0

In [6]: a.<tab>
No Warnings

File: IPython/core/completer.py (hunk 2 above)
Comment (gorogoroumaru) [reply IPython/core/completer.py]: When I changed the variable name to “interpreter”, some checks were not successful.
Why?

Comment (LucianaMarques) [reply IPython/core/completer.py]: Thanks for the change! 

About the errors, I don't know why but you can check the tests results by clicking on `details` right next to them. I will try to take a look with it too tomorrow!

Comment (LucianaMarques) [reply IPython/core/completer.py]: Which is your local python version? On one of the failed checks, there is the following message:

```
Extension error:
Could not import extension configtraits (exception: cannot import name 'l_' from 'sphinx.locale' (/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/sphinx/locale/__init__.py))
Makefile:66: recipe for target 'html' failed
make: *** [html] Error 2
make: Leaving directory '/home/travis/build/ipython/ipython/docs'
The command "if [[ "$TRAVIS_PYTHON_VERSION" == "3.7" ]] && [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
  pip install -r docs/requirements.txt
  python tools/fixup_whats_new_pr.py
  make -C docs/ html SPHINXOPTS="-W"
fi
" exited with 2.
```

Comment (gorogoroumaru) [reply IPython/core/completer.py]: Anaconda python3.7 is my local python version.
But why my local python version affects travis-CI checks?

**Comment by gorogoroumaru** [comment]
I couldn't pass the travis-ci checks, so I closed this PR and created another PR.
